sdk/trace/provider.go

```diff
@@ -99,7 +101,8 @@ func (p *Provider) GetTracer(name string) apitrace.Tracer {
 	return t
 }
 
-// RegisterSpanProcessor adds the given SpanProcessor to the list of SpanProcessors
+// RegisterSpanProcessor with the Provider. This method is safe for concurrent use.
+// TODO [freeformz]: Document what registration *means*
```

Comment:
This will be backed out. See the discussion here: https://github.com/open-telemetry/opentelemetry-go/pull/213

Reply:
removed